test/unit/css.js

```diff
@@ -1323,11 +1323,11 @@ QUnit.test( "Keep the last style if the new one isn't recognized by the browser
 	assert.equal( el.css( "position" ), "absolute", "The old style is kept when setting an unrecognized value" );
 	el = jQuery( "<div></div>" ).css( "position", "absolute" ).css( "position", " " );
 
-	// Support: Edge 14
+	// Support: Edge 14-15
```

Comment:
I'm asking who this would look like  `Chrome 1.0.0 - 1.1.0-3.1.4` or something? And I don't get what that supposed to mean, Chrome had an issue in `1.0.0` and then issue resurfaced in `1.1.0` and continue to `3.1.4`?

Then why not `Chrome 1.0.0` & `Chrome 1.1.0-3.1.4`?